### ISSUE TITLE
nan at all epochs

### ISSUE THREAD

**Issue by Spectra456** [YunYang1994/tensorflow-yolov3#289]
Hi guys. I have nan at all epochs, what is problem? I add my class to coco.names and my train.txt looks like:
`/media/spectra/GMS/PDRSV2/nomeroff-net/datasets/train/X119BM199.jpg,364,422,136,451,81`

**Comment by jubick1337** [comment]
@Spectra456 И что это было? У меня просто появляется loss nan после примерно 15 эпох.

**Comment by Spectra456** [comment]
> @Spectra456 И что это было? У меня просто появляется loss nan после примерно 15 эпох.

не знаю, я просто забил и перешел на торч.

**Comment by jubick1337** [comment]
@Spectra456 а есть хорошая реализация yolo3 на торче? если да прикрепи ссылочку, пожалуйста

**Comment by Spectra456** [comment]
> @Spectra456 а есть хорошая реализация yolo3 на торче? если да прикрепи ссылочку, пожалуйста

https://github.com/ultralytics/yolov3
